spec.md

```diff
@@ -288,6 +288,27 @@ encapsulated within the `data` attribute.
 * Constraints:
   * OPTIONAL
 
+# Privacy & Security Considerations
+Interoperability is the primary driver behind this specification, enabling such
+behavior requires information to be made available *in the clear* resulting
+in the potential for information leakage.
+
+* Context Attributes
+
+  Sensitive information *SHOULD NOT* be carried or represented in context attributes.
+  
+  CloudEvent participants *MAY* introspect and log context attributes.
```

Comment:
s/participants/consumers/  ?? maybe?  We just haven't defined "participants" yet.

Reply:
Done